.github/workflows/build.yaml

```diff
@@ -0,0 +1,23 @@
+on: [push, pull_request]
+name: build
+jobs:
+  build:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        ghc: ['8.8.3', '8.10.2']
+        os: [ubuntu-latest, macOS-latest, windows-latest]
+        exclude:
+          # a test suite dependency fails on windows/ghc8.8.3
+          - os: windows-latest
+            ghc: '8.8.3'
+      fail-fast: false
+    name: Haskell GHC ${{ matrix.ghc }} sample
```

Comment:
Missed this. Should this include the"sample"?

Reply:
Probably not :-) I fixed in the streaming PR.